### ISSUE TITLE
Seasonal ice is misclassified as a subclass of sea ice

### ISSUE THREAD

**Issue by rduerr** [ESIPFed/sweet#115]
The ENVO IRI is:  http://purl.obolibrary.org/obo/ENVO_03000071 

The issue is that seasonal ice is only there seasonally, which is not a concept that is captured by sea ice.

**Comment by lewismc** [comment]
@rduerr do you want to propose a PR for this one? I don 't want to try and then screw it up. Thanks

**Comment by lewismc** [comment]
The relevant code is at https://github.com/ESIPFed/sweet/blob/4bcfb83cabe5a12563dfcc3483df2cdf782c7e9f/src/realmCryo.ttl#L239-L242

**Comment by rduerr** [comment]
@lewismc Yes, I always was planning on coming back to all of the SWEET tickets the Harmonization group was authoring; but we were concentrating on the ENVO stuff to begin with.  I think we should start dealing with the SWEET issues as well...  That's something we can tackle when Pier isn't available to approve changed in ENVO...

**Comment by lewismc** [comment]
That’s absolutely fine @rduerr thanks for letting me know.
